content/docs/command-reference/get.md

```diff
@@ -94,7 +94,7 @@ model.pkl
 
 Note that the `model.pkl` file doesn't actually exist in the
 [root directory](https://github.com/iterative/example-get-started/tree/master/)
-of the external Git repo. Instead, the corresponding DVC-file
+of the external Git repo. Instead, the corresponding `.dvc` file
 [train.dvc](https://github.com/iterative/example-get-started/blob/master/train.dvc)
 is found, that contains `model.pkl` (in the `outs` field). DVC then
 [pulls](/doc/command-reference/pull) the file from the default
```

Comment:
Question for @ivan and/or @efiop: will `dvc get/import` continue supporting pre 1.0 DVC repos (as data sources)? Thanks

Reply:
(wrong ivan @jorgeorpinel  :) )

Reply:
yes, pre-1.0 repos should be supported

Reply:
OK then let's leave this one as-is for now. Thanks